client/src/legacy/LeftAndMain.EditForm.js

```diff
@@ -50,12 +50,10 @@ $.entwine('ss', function($){
     PlaceholderHtml: '',
 
     /**
-     * Variable: ChangeTrackerOptions
-     * (Object)
+     * This can be customised to change settings for the change tracker.
+     * See jquery.changetracker.js
      */
-    ChangeTrackerOptions: {
-      ignoreFieldSelector: '.no-change-track, .ss-upload :input, .cms-navigator :input'
-    },
+    ChangeTrackerOptions: {},
```

Comment:
I've just moved these into the default selector set for changetracker as a whole.

Sort-of unrelated to the current work - but I did notice search filters were being included in changetracking, and updating the default changetrack selectors did nothing because that set got overridden here.